features/features_test.go

```diff
@@ -378,59 +378,93 @@ func (s *featureSuite) TestAll(c *C) {
 	defer st.Unlock()
 	tr := config.NewTransaction(st)
 
+	fakeFeature := features.SnapdFeature(features.NumberOfFeatures())
+	is, why := fakeFeature.IsSupported()
+	c.Check(is, Equals, true)
+	c.Check(why, Equals, "")
+
+	restore := features.MockKnownFeaturesImpl(func() []features.SnapdFeature {
+		return []features.SnapdFeature{fakeFeature}
+	})
+	defer restore()
+
+	features.FeatureNames[fakeFeature] = "fake-feature"
+	defer func() {
+		delete(features.FeatureNames, fakeFeature)
+	}()
```

Comment:
Small nitpick, This could be moved to mock function in `export_test.go` where `featureNames` is backed up and restored.

```go
restore = features.MockFeatureNames(map[SnapdFeature]string{fakeFeature: "fake-feature"})
defer restore()
```